saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/nlp/BaseTypes/SegmentRelation.java

```diff
@@ -0,0 +1,35 @@
+package edu.illinois.cs.cogcomp.saulexamples.nlp.BaseTypes;
+
+/**
+ * Created by Umar Manzoor on 02/01/2017.
+ */
+public class SegmentRelation {
+    private String imageID;
+    private int segment_ID1;
+    private int segment_ID2;
```

Comment:
It's rare to use underscores for naming in java community it's more like C-Style, so I prefer `segmentId1` . If there are two segments, you can name them `firstSegmentId` and `secondSegmentId`